Add Project Loved: May 2022 news post (#7356)
ppy/osu-wiki: news/2022-05-21-project-loved-may-2022.md

---
layout: post
title: "Project Loved: May 2022"
date: 2022-05-21 12:00:00 +0000
---

> May you come vote on this month's set of maps nominated in Project Loved?

Author: it was meant to be the world's lowest effort joke on "May" but I think it's so bad that the low effort sarcasm doesn't even come thru

> May you come vote on this month's set of maps nominated in Project Loved?

Author: 2 got a nice rhythm to it

![](/wiki/shared/news/banners/project-loved.jpg)

For those unfamiliar, Project Loved hosts monthly votings to decide which nominated maps get added to the [Loved category](https://osu.ppy.sh/beatmapsets?s=loved). You can click on each of the map's banners in this post to go to its poll on the forums, or visit [the forum](https://osu.ppy.sh/community/forums/120) for a list of all of the polls. Polls will be open for 10 days, so make sure to cast your votes in time! More info can be found on the [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh)!

> First, thanks for your participation in last month's voting—summaries of the results can be found in the following forum posts:

Author: take away my unspaced em dashes, and what do i have left

- [osu!](https://osu.ppy.sh/community/forums/posts/8615510)
- [osu!taiko](https://osu.ppy.sh/community/forums/posts/8615509)
- [osu!catch](https://osu.ppy.sh/community/forums/posts/8615508)
- [osu!mania](https://osu.ppy.sh/community/forums/posts/8615507)

After over two years of dedicated captain-ship to Project Loved, this month [Spectator](https://osu.ppy.sh/users/702598) decided to step down from his role in osu!catch. Make sure there's no shortage of thanks going his way if you appreciated his work!

Similarly, [hypercyte](https://osu.ppy.sh/users/9155377) spent nearly two years contributing his effort to Loved by correcting metadata of nominated maps, but recently retired from both here and the NAT. [radar](https://osu.ppy.sh/users/7131099) and [Yasuho](https://osu.ppy.sh/users/8458835) began work in the same area this month to fill the gap!

> Another development you may have been wondering about is that survey Project Loved ran at the end of February. While there are still no concrete plans made based on the results, [you can view those results on the Project loved website](https://loved.sh/survey/community-feedback-february-2022). I'm still working on options for filtering and comparing the data there; if there's anything you have a hard time finding from the existing graphs, let me know in the `#osu-loved` channel of the [osu!dev Discord server](https://discord.gg/ppy).

Author: my high school english teachers would be disappointed in me if I switched an active verb to a passive one so just fixing first part

A popular set among multiplayer goers, and occasionally in tournament play, Jean-Michel Jr and company have put together a challenging cutstream set that does not let off the gas for three and a half minutes. It is full of quad and double cutstream patterns set at 175 BPM, accompanied by a dusting of technical slider mapping. Most of this five difficulty set is rated at around five to six and a half stars, all with their own take on the technical stream theme. Each of them culminate into a huge drop section at the end of the song, and the complex shapes all step up a gear to match.

This is an absolute must-have for anyone who enjoys cutstream-heavy mapping, and what's not to like? Give us your vote on whether we should add this set to Loved!

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/74115.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Hige Driver - ukigumo](https://osu.ppy.sh/beatmapsets/74115#osu) by [DsG](https://osu.ppy.sh/users/498862)\
118.75 – 178.12 BPM, 3:29 | 4.87★, 4.87★, 6.82★, 6.82★, 6.82★\
*description written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*

> As one of the most classic graveyarded chiptune songs, *ukigumo* stands out as a breath of fresh air. The mapping style is as innocent as 2013 gets, featuring mostly aim patterns as it builds up to a flow aim climax at the end. The difficulty spread of this mapset is also quite unique—there are 5 difficulties total, each with different ARs. While this may sound quite common, this mapset takes it a step further and has slowed down versions of the map to be played with DT such that the original map can be played at AR10.3 and AR11. For all of you high AR enthusiasts out there, this mapset is definitely your calling, but even if you're not built for high AR, this mapset is nothing but innocent fun for everyone.

Author: "ARs" is p normal to say

270 BPM, 2:32 | 6.53★, 6.69★, 7.00★\
*description written by [KuroKuroKuro](https://osu.ppy.sh/users/11931563)*

*Nageki no Hana (Slax Remix)* is a very popular map among Taiko players. I often see various Taiko players pick and play this map in multi mode. The fact that it is loved by such a wide variety of players must mean that the map is that good. The BPM is fast (270), but there are many basic patterns, so this is a good speed map for intermediate to advanced players to practice. 1/6 is not included at all, and only 1/4 is used, so it is recommended for those who are not good at fast maps and want to improve. If you're looking for a way to hone your Taiko skills, why not play this map?

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/529279.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[goreshit - loli rockers](https://osu.ppy.sh/beatmapsets/529279#taiko) by [uone](https://osu.ppy.sh/users/5321719)\
194 BPM, 2:33 | 6.08★\
*description written by [Horiiizon](https://osu.ppy.sh/users/8071438)*

> Taiko players in the house tonight, *loli rockers* by uone is up for voting this month! The map is filled with comfortable patterns in the style uone is well known for; it's also a lot easier than the typical Loved maps we see nowadays, featuring only short 1/4 bursts and a few 1/6 in the entire map. However, that is not to say the map is a breeze, as there are finishers and slightly unconventional 1/6 patterns to look out for. Despite being on the easier side for Loved and despite the map not containing any SV changes, *loli rockers* fits the song well and is very fun and engaging from start to finish. Have a good time!

Author: the unanimous usage of "Taiko" makes me think that's just the natural way to write it so imma keep that part (and where it comes up in other comment)

> Taiko players in the house tonight, *loli rockers* by uone is up for voting this month! The map is filled with comfortable patterns in the style uone is well known for; it's also a lot easier than the typical Loved maps we see nowadays, featuring only short 1/4 bursts and a few 1/6 in the entire map. However, that is not to say the map is a breeze, as there are finishers and slightly unconventional 1/6 patterns to look out for. Despite being on the easier side for Loved and despite the map not containing any SV changes, *loli rockers* fits the song well and is very fun and engaging from start to finish. Have a good time!

Reviewer: ok maybe, but why capitalise it everywhere?

> Taiko players in the house tonight, *loli rockers* by uone is up for voting this month! The map is filled with comfortable patterns in the style uone is well known for; it's also a lot easier than the typical Loved maps we see nowadays, featuring only short 1/4 bursts and a few 1/6 in the entire map. However, that is not to say the map is a breeze, as there are finishers and slightly unconventional 1/6 patterns to look out for. Despite being on the easier side for Loved and despite the map not containing any SV changes, *loli rockers* fits the song well and is very fun and engaging from start to finish. Have a good time!

Author: ¯\\\_(ツ)\_/¯

243 BPM, 5:58 | 8.28★, 8.71★\
*description written by [Kasumi-sama](https://osu.ppy.sh/users/6177263)*

Resident osu!catch mapper Minato Yukina makes once again his return to the Loved section, with potentially his 20th Loved beatmap, *Contrapasso -purgatorio-*, which was mapped for [AxS' 4th edition](https://osu.ppy.sh/community/forums/topics/822420)'s Grand Finals tiebreaker. This map takes any osu!catch player to one hellish ride, with the first two kiai sections riddled with relatively hard antiflow patterns, jumps and bursts, and the map does not stop the intensity it pulls out in the slightest, even up to the very end, with the nastiest burst right at the end, ready to kill players.

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1107782.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Chihiro Nakamura - Kasaneteku](https://osu.ppy.sh/beatmapsets/1107782#fruits) by [Unlucky_w](https://osu.ppy.sh/users/4820793)\
147 BPM, 3:43 | 6.71★\
*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*

This month we have a new face in the Loved category: Unlucky_w, formerly known as FastYoshi. And oh boy, this map is really interesting.

> First of all, this is perhaps one of the catchiest songs ever, and for that reason a good selling point for restarting the map over and over again until you can't get it out of your head. But not only that; the map itself is also catchy in a sense. It flows really well and fits the song nicely, highlighting the key parts of it. It also plays differently from most of the other maps of this day and age but in a really good way. It is not an easy map, but a very doable one with some practice.

Reviewer: ```suggestion
First of all, this is perhaps one of the catchiest songs ever, and for that reason a good selling point for restarting the map over and over again until you can't get it out of your head. But not only that, the map itself is also catchy in a sense. It flows really well and fits the song nicely, highlighting the key parts of it. It also plays differently from most of the other maps of this day and age but in a really good way. It is not an easy map, but a very doable one with some practice.
```

Truly a hidden gem, you should definitely give it a try (and get addicted to the song along the way).

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/945848.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Masayoshi Minoshima - Bad Apple!! feat. nomico (Camellia's "Bad Psy!!" Remix)](https://osu.ppy.sh/beatmapsets/945848#fruits) by [Kukkai](https://osu.ppy.sh/users/7811952)\
160 BPM, 4:25 | 7.08★\
*description written by [Sartan](https://osu.ppy.sh/users/4100941)*

> Right, let's keep this one short. You know Kukkai by now. No, actually, you love Kukkai by now. We know you do. So here's another one of their maps! *Bad Apple* is one of the most popular songs in the osu! community, so a Camellia remix (of all things) of it will never be received in a bad manner by you lovely people. This map is pure quality, despite the fact that it has some quirky streaming patterns in combination with some of the drops in the song. But hey, that's what Loved is for, right?! We are sure you'll like it. The only thing to note is that this is a relatively difficult map (though not extreme), so only fairly advanced players allowed for this one!

Reviewer: ```suggestion
Right, let's keep this one short. You know Kukkai by now. No, actually, you love Kukkai by now. We know you do. So here's another one of their maps! *Bad Apple* is one of the most popular songs in the osu! community, so a Camellia remix (of all things) of it will never be received in a bad manner by you lovely people.

This map is pure quality, despite the fact that it has some quirky streaming patterns in combination with some of the drops in the song. But hey, that's what Loved is for, right?! We are sure you'll like it. The only thing to note is that this is a relatively difficult map (though not extreme), so only fairly advanced players allowed for this one!
```

> Now, don't be such a bad apple and go vote! Wait I mea-

Reviewer: ```suggestion
Now, don't be such a bad apple and go vote! Wait I mea—
```

[UNDEAD CORPORATION - The Empress](https://osu.ppy.sh/beatmapsets/358342#mania) by [\_underjoy](https://osu.ppy.sh/users/2235750)\
270 BPM, 4:20 | 6.58★\
*description written by [-NoName-](https://osu.ppy.sh/users/10400730)*

Next up is \_underjoy's chart of *The Empress*!

If you are familiar with \_underjoy's [ranked set](https://osu.ppy.sh/beatmapsets/315435) of *The Empress scream off ver*, you will also quickly become familiar with his longer chart of *The Empress*. This chart is packed to the brim with relentless stamina-draining patterns at 270 BPM. If you ever wanted to give a longer version of the ranked set a leaderboard, now is your chance.

Set this chart of *The Empress* free from the deepest nightmare that is the graveyard and cast your vote!

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1098451.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Senya - Melody Blooming in the Earth](https://osu.ppy.sh/beatmapsets/1098451#mania) by [\[Crz\]Rachel](https://osu.ppy.sh/users/8369409)\
132 BPM, 4:32 | 5.80★, 6.27★, 6.76★, 7.26★, 7.74★\
*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*

> It took some months for them to come back into the spotlight, but they make a strong reappearance in the Loved polls: \[Crz\]Rachel is yet again offering the community one of their well-known chordjack charts, this time *Senya's Melody Blooming in the Earth*!

Reviewer: ```suggestion
It took some months for them to come back into the spotlight, but they make a strong reappearance in the Loved polls: \[Crz\]Rachel is yet again offering the community one of their well-known chordjack charts, this time Senya's *Melody Blooming in the Earth*!
```

> To be frank, it has been fairly difficult to choose something from them as they have tons of charts Loved by the community. However, we believe that the rate accessibility of this chart in particular might be more enjoyed across the board! As a player, you should expect heavily anchored sections, a recurring motif in the chart. These seem to spice the experience however, as getting to control them is what's going to give you an edge.

Reviewer: ```suggestion
To be frank, it has been fairly difficult to choose something from them as they have tons of charts loved by the community. However, we believe that the rate accessibility of this chart in particular might be more enjoyed across the board! As a player, you should expect heavily anchored sections, a recurring motif in the chart. These will surely spice up the experience, so get ready to constantly be on edge getting used to controlling them!
```

> Other than that, not much else can be said. Just a gamer tip; try hitting the note. It tends to work fairly well.

Reviewer: ```suggestion
Other than that, not much else can be said. Just a gamer tip: try hitting the note. It tends to work fairly well.
```

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1012105.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Rish feat. Choko - Bi](https://osu.ppy.sh/beatmapsets/1012105#mania) by [Raveille](https://osu.ppy.sh/users/1388767)\
169 BPM, 1:54 | 4.90★\
*description written by [Paturages](https://osu.ppy.sh/users/1375479)*

> Contrary to popular assumption(?), this song is not about the "B" of LGBT, nor is it about the letter "B" itself, the insect known as "bee" or its dedicated movie, or even the 🅱️ emoji. There's only one way to find out whether or not Unicode emojis work in news posts, let's see how this goes.

Reviewer: i mean what do you expect, of course it will

```suggestion
Contrary to popular assumption(?), this song is not about the "B" of LGBT, nor is it about the letter "B" itself, the insect known as "bee" or its dedicated movie, or even the 🅱️ emoji.
```

Author: idk either but this was captains joke so imma keep it

> *Bi*, or *び*, has one syllable more prevalent than the others throughout all of its length. You can go ahead and guess which one. In typical Raveille fashion, this map showers you with a stout density of long notes and will test your ability to read through them before testing your ability to actually play them.

Reviewer: italics on hiragana does literally nothing

```suggestion
*Bi*, or び, has one syllable more prevalent than the others throughout all of its length. You can go ahead and guess which one. In typical Raveille fashion, this map showers you with a stout density of long notes and will test your ability to read through them before testing your ability to actually play them.
```

Author: :thinking: it does for me

> As a featured map in the [MWC 2020 quarterfinals](/wiki/Tournaments/MWC/2020_4K), this would add to the mid-to-high level of density oriented long note catalogue of the Loved section. Speaking of which, this is not Raveille's first long note Loved entry, and some of his prouder charts like [Delrio](https://osu.ppy.sh/beatmapsets/1251408) or [Mare Maris](https://osu.ppy.sh/beatmapsets/917906) have even gone as far as getting ranked! The popularity aspect of *Bi*, however, had us interested in including it in the Loved section.

Reviewer: ```suggestion
As a featured map in the [MWC 2020 quarterfinals](/wiki/Tournaments/MWC/2020_4K), this would add to the mid-to-high level of density-oriented long note catalogue of the Loved section. Speaking of which, this is not Raveille's first long note Loved entry, and some of his prouder charts like [Delrio](https://osu.ppy.sh/beatmapsets/1251408) or [Mare Maris](https://osu.ppy.sh/beatmapsets/917906) have even gone as far as getting ranked! The popularity aspect of *Bi*, however, had us interested in including it in the Loved section.
```

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/789869.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Camellia - liquated](https://osu.ppy.sh/beatmapsets/789869#mania) by [-MysticEyes](https://osu.ppy.sh/users/6253266)\
140 BPM, 4:22 | 4.65★\
*description written by [Paturages](https://osu.ppy.sh/users/1375479)*

For a decent amount of time, -MysticEyes has been a reliable source of maps of both convincing quality and adventurous ambition. With no less than 15 ranked mapsets and [one Loved LN entry](https://osu.ppy.sh/beatmapsets/800341), -MysticEyes has experimented and played around with a lot of different mapping styles and skillset focuses.

As such, *liquated* is a surprising entry. Its focus resides in something that has been a bit popular lately in the ranking section: stream dumps. You would not expect to find such a map in -MysticEyes's repertoire, especially not in 2018. At that point in time, it would have been less imaginable to rank this type of map, and as much as we would like to see this resurface there in 2022, this map will have to remain a testament of the past.

> Despite not being featured in any major way and being more than 4 minutes long, it has nonetheless amassed a fair amount of spontaneous popularity, enough for us to consider this for the Loved section. Give this one a try: you will be delightfully surprised, as this plays quite smoothly throughout all its length. Don't forget to hit us with a vote afterwards!

Reviewer: ```suggestion
Despite not being featured in any major way and being more than 4 minutes long, it has nonetheless amassed a fair amount of spontaneous popularity, enough for us to consider this for the Loved section. Give this one a try — you will be delightfully surprised, as this plays quite smoothly throughout all its length. Don't forget to hit us with a vote afterwards!
```

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1428709.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[C-show - GIMME DA BLOOD](https://osu.ppy.sh/beatmapsets/1428709#mania) by [H1Pur](https://osu.ppy.sh/users/15756120)\
220 BPM, 2:11 | 2.75★, 2.75★\
*description written by [Paturages](https://osu.ppy.sh/users/1375479)*

> H1Pur finally makes his entrance to the Loved section! As an up and coming mapper who has recently been contributing to a number of tournaments with tasteful SV mapping, starting with [SVSE](https://osu.ppy.sh/community/forums/topics/1313374), he is also perhaps the best known enthusiast of Amelia Watson we know in the osu!mania community. Feel free to prove us wrong in the voting thread.

Reviewer: ```suggestion
H1Pur finally makes his entrance to the Loved section! As an up and coming mapper who has recently been contributing to a number of tournaments with tasteful SV mapping, starting with [SVSE](https://osu.ppy.sh/community/forums/topics/1313374), he is also perhaps the best known enthusiast of the Amelia Watson we know in the osu!mania community. Feel free to prove us wrong in the voting thread.
```

Author: "we know in the osu!mania community" refers to H1Pur

*GIMME DA BLOOD* made its first appearance in SVSE as a custom map, and was then brought back again in [MWC 2021](/wiki/Tournaments/MWC/2021_4K). It's an honest and accessible reading-oriented early-game SV pick that you may find yourself headbanging to instead of actually learning. Watch out for those BPM changes, they can be a bit finicky to learn!

Look, I know that you can only vote once, but: *One for the blood, two for the life... GIMME DA VOTES!*

[![](/wiki/shared/news/2022-05-21-project-loved-may-2022/1555979.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[nowisee - Ko Inu](https://osu.ppy.sh/beatmapsets/1555979#mania) by [ruka](https://osu.ppy.sh/users/6117525), [emO_Oticon](https://osu.ppy.sh/users/12064488), [Lung_P](https://osu.ppy.sh/users/8271308) and [sankansuki](https://osu.ppy.sh/users/2877926)\
176 BPM, 4:22 | 6.71★, 7.27★, 9.31★\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

ruka has become a household name in the 7K LN mapping community. We decided to nominate another crowd-favourite of theirs, *Ko Inu*!

> This mapset consists of a LN-oriented 4K GD by Lung_P, hybrid style 7K GD by sankansuki, and ruka's collab with emO_Oticon which features very dense LN patterning, especially in the second half. Combined with the high intensity of the song, this mapset is a very memorable experience. If you share similar thoughts, consider voting for *Ko Inu*!

Reviewer: gonna avoid arguing over "a" and "an" again

```suggestion
This mapset consists of a highly LN-oriented 4K GD by Lung_P, a hybrid-style 7K GD by sankansuki, and ruka's collab with emO_Oticon which features very dense LN patterning, especially in the second half. Combined with the high intensity of the song, this mapset is a very memorable experience. If you share similar thoughts, consider voting for *Ko Inu*!
```
